packages/index-model/lib/model.js

```diff
@@ -102,6 +102,12 @@ var IndexModel = Model.extend({
         return !!this.extra.collation;
       }
     },
+    clustered: {
+      deps: ['extra'],
+      fn: function() {
+        return !!this.extra.clustered;
+      }
+    },
     type: {
       deps: ['geo', 'hashed', 'text', 'wildcard'],
```

Comment:
Should `clustered` also be in this list? I might be misunderstanding how these are used in ampersand.

Reply:
Probably. Added.